docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/managing-channels.md

```diff
@@ -1,6 +1,6 @@
 # Managing Channels
```

Comment:
Is there any documentation on channel-level configuration (i.e. managing a channel)?